content/docs/start/experiments-trail/experiments-1.md

```diff
@@ -0,0 +1,334 @@
+the files manually. We use this feature to set the convolutional units in
+`train.py`.
+
+```dvc
+$ dvc exp run --set-param model.conv_units=24
+...
+Reproduced experiment(s): exp-6c06d
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+When you run `dvc exp run` with `--set-param`, it updates the parameter file. We
+can see the effect of it by looking at the diff.
+
+```dvc
+$ git diff params.yaml
+```
+
+```git
+-model:
+-  conv_units: 16
++model:
++  conv_units: 24
+```
+
+## Run multiple experiments in parallel
+
+Instead of running the experiments one-by-one, we can define them to run in a
+batch. This is especially handy when you have long running experiments.
+
+We add experiments to the queue using the `--queue` option of `dvc exp run`. We
+also use `-S` (`--set-param`) to set a value for the parameter.
+
+```dvc
+$ dvc exp run --queue -S model.conv_units=32
+Queued experiment '6518f17' for future execution.
+$ dvc exp run --queue -S model.conv_units=64
+Queued experiment '30eb9b2' for future execution.
+$ dvc exp run --queue -S model.conv_units=128
+Queued experiment 'ac66940' for future execution.
+$ dvc exp run --queue -S model.conv_units=256
+Queued experiment '8bb6049' for future execution.
+```
+
+Next, run all (`--run-all`) queued experiments in parallel (using `--jobs`):
+
+```dvc
+$ dvc exp run --run-all --jobs 2
+```
+
+It runs all the experiments, with the specified number of processes in parallel.
+
+## Comparing experiments
+
+The experiments are run several times with different parameters. We use
+`dvc exp show` to compare all of these experiments. This command presents the
+parameters and metrics produced in experiments in a nicely formatted table.
+
+```dvc
+$ dvc exp show
+```
+
+![](/img/start-dvc-exp-show-210704.png)
+
+By default it shows all the parameters and the metrics with the timestamp. If
+you have large number of parameters, metrics or experiments, this may lead to a
+cluttered view. You can limit the table to specific metrics, or parameters, or
+hide the timestamp column with `--include-metrics`, `--include-params`, or
+`--no-timestamp` options of the command, respectively.
+
+```dvc
+$ dvc exp show --no-timestamp \
+  --include-params model.conv_units --include-metrics acc
+```
+
+![](/img/start-dvc-exp-show-no-timestamp-210704.png)
+
+## Persisting experiments
+
+After selecting a experiments from the table, you can commit the hyperparameters
+and other dependencies that produced this successful experiment to your Git
+history.
+
+`dvc exp apply` brings back all specific artifacts and parameters from the
+experiment to the <abbr>workspace</abbr>.
+
+```dvc
+$ dvc exp apply exp-cb13f
+Changes for experiment 'exp-cb13f' have been applied to your current workspace.
+```
+
+We can see the changes in the repository and commit them to Git.
+
+```dvc
+$ git diff
```

Comment:
Again here, probably no need for `diff`. Maybe `status`?
````suggestion
We can see the changes in the repository and commit them to Git.

```dvc
$ git status
...
````

Reply:
OR just don't mention seeing the repo changes, just focus on `git commit`.

Reply:
How can the user know what changed _in files_ without `git diff`? All experiments supposedly change the same files, that is `metrics.json` here, the reason we are selecting a particular experiment is that these metrics and associated files are better than others.

Reply:
If the user knows Git they will run `git diff` if they want to. The terminal sample above doesn't even show the output of `git diff` so I'm not sure what's being illustrated. I don't think we should document Git per se.

> All experiments supposedly change the same files, that is metrics.json here

You could do `exp/metrics diff` here if needed. Not sure whether it is (needed).